### ISSUE TITLE
gopls: automated issue report (initialization)

### ISSUE THREAD

**Issue by keenangebze** [golang/vscode-go#1314]
gopls version: v0.6.4
gopls flags: 
extension version: 0.23.1
environment: Visual Studio Code linux
initialization error: Error: JSON RPC parse error: json: cannot unmarshal number into Go struct field RenameClientCapabilities.prepareSupportDefaultBehavior of type bool: JSON RPC parse error
manual restart count: 0
total start count: 1

ATTENTION: PLEASE PROVIDE THE DETAILS REQUESTED BELOW.

Describe what you observed.

Just updated gopls.

I'm initiating the workspace inside GVM
```
gvm use go1.12.4
cd $GOPATH/src/github.com/company/
code repo
``` 

repo is a link to:
```
ls -lah
lrwxrwxrwx  1 user group   43 Feb 15 14:24 repo -> /home/user/go/src/github.com/company/repo
```

When opening vscode got this alert:
```
The gopls server failed to initialize Would you like to report a gopls issue on GitHub?
You will be asked to provide additional information and logs, so PLEASE READ THE CONTENT IN YOUR BROWSER.
```

It seems it only error when I used `go1.12.4` in gvm, when I tried `go1.14.4` it does not get the alert.
I update the gopls when opening workspace on `go1.14.4` 

<pre>Starting client failed
  Message: JSON RPC parse error: json: cannot unmarshal number into Go struct field RenameClientCapabilities.prepareSupportDefaultBehavior of type bool: JSON RPC parse error
  Code: -32700 </pre>

OPTIONAL: If you would like to share more information, you can attach your complete gopls logs.

NOTE: THESE MAY CONTAIN SENSITIVE INFORMATION ABOUT YOUR CODEBASE.
DO NOT SHARE LOGS IF YOU ARE WORKING IN A PRIVATE REPOSITORY.

```
[Error - 10:27:40 AM] Starting client failed
  Message: JSON RPC parse error: json: cannot unmarshal number into Go struct field RenameClientCapabilities.prepareSupportDefaultBehavior of type bool: JSON RPC parse error
  Code: -32700 
gopls: failed reading header line: &{%!w(string=EOF)}
```

**Comment by hyangah** [comment]
Thanks for the detailed info. @keenangebze 

The root cause of the original issue is #1300.
You will need a recent version of gopls (v0.6.6). It's possible that when you were using go1.12.4, an older version of gopls (v0.6.4) was picked up by gvm. Please update gopls. 

BTW, go1.12 is a no longer supported go version in go project and (gopls still may compile, but there is no guarantee). Is there any reason you need to work with `go1.12.4`?

**Comment by keenangebze** [comment]
Thanks for the link @hyangah, solved the issue as instructed in #1300 
```
gvm use go1.12.4
cd $GOPATH/src/github.com/company
GO111MODULE=on go get golang.org/x/tools/gopls@latest
code repo
```
We have both old and new repository that uses multiple version of golang. 
I guess updating the go version for the old repository takes additional effort for the team (moving from dep to go mod).
I'll keep that in mind and raise it to my team, thank you

**Comment by hyangah** [comment]
Thanks. I will close this issue. I hope we can address #1300 soon.
